src/client/components/Button/ButtonComponent.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ButtonComponent.styles.css';
+
+export default function ButtonComponent({ title, backgroundColor, onClick }) {
+  return (
+    <div className="button-component">
+      <button
+        type="button"
+        onClick={onClick}
+        className="button button--primary button--medium"
+        style={{ backgroundColor }}
```

Comment:
If you want to pass in the classes as parameters to the button, probably you do not need to have background color.

Reply:
okay